nucypher/cli/painting/staking.py

```diff
@@ -267,3 +275,50 @@ def paint_min_rate(emitter, staker):
     ~ Previously set ....... {prettify_eth_amount(raw_minimum)}
     ~ Effective ............ {prettify_eth_amount(minimum)}"""
     emitter.echo(rate_payload)
+
+
+def paint_staking_rewards(stakeholder, blockchain, emitter, past_periods, staking_address, staking_agent):
+    if not past_periods:
+        reward_amount = stakeholder.staker.calculate_staking_reward()
+        emitter.echo(message=TOKEN_REWARD_CURRENT.format(reward_amount=round(reward_amount, TOKEN_DECIMAL_PLACE)))
+        return
+
+    economics = stakeholder.staker.economics
+    seconds_per_period = economics.seconds_per_period
+    current_period = staking_agent.get_current_period()
+    from_period = current_period - past_periods
+    latest_block = blockchain.client.block_number
+    from_block = estimate_block_number_for_period(period=from_period,
+                                                  seconds_per_period=seconds_per_period,
+                                                  latest_block=latest_block)
+
+    argument_filters = {'staker': staking_address}
+    event_type = staking_agent.contract.events['Minted']
+    entries = event_type.getLogs(fromBlock=from_block,
+                                 toBlock='latest',
+                                 argument_filters=argument_filters)
+
+    rows = []
+    rewards_total = NU(0, 'NU')
+    for event_record in entries:
+        event_block_number = int(event_record['blockNumber'])
+        event_period = event_record['args']['period']
+        event_reward = NU(event_record['args']['value'], 'NuNit')
+        timestamp = blockchain.client.get_block(event_block_number).timestamp
+        event_date = maya.MayaDT(epoch=timestamp).local_datetime().strftime("%b %d %Y")
+        rows.append([
+            event_date,
+            event_block_number,
+            int(event_period),
+            round(event_reward.to_tokens(), TOKEN_DECIMAL_PLACE),
```

Comment:
Don't need `.to_tokens()`
```suggestion
            round(event_reward, TOKEN_DECIMAL_PLACE),
```